### PR TITLE
Change error message under `add_people`

### PR DESCRIPTION
**Pull request by yryuvraj**
Issue assigned in today's standup. 

Changed Output : 
image

### CODE REVIEW

**Comment by bhanuvrat** [comment]
Don't we show "email has been taken" message when people are signing up on the application?
Is this a good message in this context?

**Comment by yryuvraj** [comment]
You make a good point. Nupur asked me to do this, I did not think it through while implementing.

We can change it to "Email has already been added to the list" for better clarity, maybe? Will discuss with Nupur in tomorrow's meeting.

File: docs/sender_profile/validation.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+### `username`
+- Must be at least 3 characters long.
+- Cannot be more than 100 characters long.
+
+### `password`
+- Must be at least 6 characters long.
+- Cannot be more than 25 characters long.
```
Comment (bhanuvrat) [inline docs/sender_profile/validation.md:7]: 255 chars upper limit.

File: docs/sender_profile/validation.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### `smtp_host`
+- Must be at least 5 characters long.
+- Cannot be more than 100 characters long.
+- Must be a valid host name (e.g., `example.com`).
+- It can either be a domain name or an ip address
+
+### `smtp_port`
+- Must be a valid port number (only digits).
```
Comment (bhanuvrat) [inline docs/sender_profile/validation.md:16]: between 0 and 65535

Comment (yryuvraj) [reply docs/sender_profile/validation.md:16]: I implemented it for 65535, documented it wrong 🤦🏻
Will make the necessary changes.

**Comment by yryuvraj** [comment]
@bhanuvrat made the necessary changes, when you merge this, I'll push the implementation of the validation. It's almost ready on my local machine.

**Review by bind-ritu1211 — COMMENTED** [review]
lgtm

**Pull request merged**